docs/how-to/capture-logs-from-files.md

```diff
@@ -0,0 +1,67 @@
+# How to capture logs from files
+
+Pebble stores the most recent `stdout` and `stderr` from each service. However, some
+applications don't log to `stdout` or `stderr`. Instead, they write logs directly to
+files such as `access.log`, `audit.log`, or `error.log`.
+
+This guide demonstrates how to use the `tail` command to print log files to `stdout`,
+so that Pebble can capture the logs. The `tail` command is provided by the `coreutils`
+package.
+
+## Define a layer
+
+To use `tail`, we'll define a layer that has a main service (`foo`) and a helper service
+for `tail`:
+
+```{code-block} yaml
+  :emphasize-lines: 5, 7, 10
+
+services:
+  foo:
+    command: foo
+    startup: enabled
+    requires:
+      - foo-error-log
+    after:
```

Comment:
What happens if it's the first time the app starts and the log file doesn't exist yet?

Reply:
`tail -F` handles that.
- tail will log one line that file is missing
- when files is created, it will start tailing